src/blocks/cards.css

```diff
@@ -0,0 +1,103 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  padding: 32px 16px 16px;
+  width: 200px;
+}
+
+.card__content {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.card__page {
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+}
+
+.card__text {
+  font-family: "Roboto-Medium", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin-bottom: 4px;
+  color: var(--color-text);
+}
+
+.card__product-code {
+  font-family: "Roboto", sans-serif;
```

Comment:
move font-family to html tag to avoid using this rule each time

Reply:
https://github.com/mate-academy/layout_product-cards/pull/2976/files#r1181794006
not fixed

DRY, just move `font-family` to `html` tag selector